client/grpc-gateway/config.json

```diff
@@ -0,0 +1,90 @@
+{
+  "swagger": "2.0",
```

Comment:
If i'm not mistaken, there's a new version 3, rebranded to OpenAPI. Why don't we use v3 instead? Maybe the merging issue will not even be relevant anymore.

Edit: that's why https://github.com/grpc-ecosystem/grpc-gateway/issues/441. Keeping this comment for an FYI.